pkg/tools/assessment.go

```diff
@@ -151,6 +154,22 @@ func writeResultValues(w io.Writer, result *Result) {
 
 func addCustomPolicyMetadata(result *Result, metadata map[string]string) {
 	for k, v := range metadata {
-		result.AddValue(fmt.Sprintf("CUSTOM_POLICY_%s", k), v)
+		key := fmt.Sprintf("CUSTOM_POLICY_%s", k)
+		if key == "CUSTOM_POLICY_UPLOAD_PATH" {
+			log.Infof(fmt.Sprintf("Custom policy reference: %s", strings.TrimPrefix(v, "custom_policies/")))
```

Comment:
What is the user being informed of here? Might this be plainer as `"Using custom policy path: %s`

Reply:
It's pretty irrelevant to the user and we don't want to disclose that it is a path to anything. It's being logged at info level  to ensure it is written to scalyr. That way in the scalyr logs we will see this reference and be able to determine the path to the storage bucket for the policies used by a scan.

Reply:
I mistook this log entry for a CLI output, my bad